cfme/containers/image.py

```diff
@@ -63,6 +63,20 @@ def exists(self):
     def sha256(self):
         return self.id.split('@')[-1]
 
+    @property
+    def last_scan_attempt_on(self):
+        """ Get 'Last Scan Attempt On' property for OCP Image.
+
+            Returns:
+                datetime or None.
+        """
+        try:
+            return (self.appliance.rest_api.collections.container_images.
+                    get(name=str(self.name)).last_scan_attempt_on)
```

Comment:
Optional:
`name` is pointing to `image` name and It will be `string` only. In place of converting here better to specify in Class `doc-string`

Reply:
Added comment explaining the use of str() here